cmd/security-agent/api/agent/agent.go

```diff
@@ -51,6 +54,26 @@ func (a *Agent) SetupHandlers(r *mux.Router) {
 	r.HandleFunc("/config/list-runtime", a.settings.ListConfigurable).Methods("GET")
 	r.HandleFunc("/config/{setting}", a.settings.GetValue).Methods("GET")
 	r.HandleFunc("/config/{setting}", a.settings.SetValue).Methods("POST")
+	r.HandleFunc("/workload-list/short", func(w http.ResponseWriter, r *http.Request) {
+		workloadList(w, false, a.wmeta)
+	}).Methods("GET")
+	r.HandleFunc("/workload-list/verbose", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Rather than having two separate endpoints, could we use an URL params to decide if is the output should use the verbose mode or not? 

Similar to what we do in https://github.com/DataDog/datadog-agent/blob/8e4391bc55921bbd0e69dafb8b5da1be2e1cc979/comp/api/api/apiimpl/internal/agent/agent.go#L148

Reply:
I actually copied most of this code from the process agent https://github.com/DataDog/datadog-agent/blob/6b3d202f342cb84b0ab329b4287a792dd77647cd/cmd/process-agent/api/server.go#L48 but yeah I can use an URL param for sure

Reply:
should be fixed in latest commit

Reply:
@paulcacheux Thank you so much for taking the time to change the code. 

I know it might be a stretch, but would you be willing to change the process agent code as well so we follow the same pattern in both agent binaries?

Reply:
I'm not sure if the short and verbose version were never actually used. I've looked in the code and the only use of verbose for `workload-list` used URL params

https://github.com/DataDog/datadog-agent/blob/6bf355e35ff17e5da487ae18a58d314ca5d21955/pkg/flare/archive.go#L339

Reply:
Hmm yeah I would prefer not touching their code, the risk + the delay of asking for another review etc

Reply:
short and verbose are usable from the subcommands CLI, so it may be used by CLI users (you have a param `--verbose` to switch the mode)